### PR TITLE
Commit action docs

### PR DESCRIPTION
**Pull request by mfocko**
Merge with packit/packit#2070

### CODE REVIEW

File: docs/configuration/actions.md [hunk 1]
```diff
@@ -123,6 +124,40 @@ actions:
   - bash -c "sed -i -r \"s/Release:(\s*)\S+/Release:\1${PACKIT_RPMSPEC_RELEASE}%{?dist}/\" my_specfile_path"
 ```
 
+### `commit-message`
+
+For your own debugging purposes we allow arbitrary output before outputting the
+actual commit message. Your commit message **needs** to be separated by the
+following separator (on a separate line):
+
+    ---%<--- snip ---%<--- here ---%<---
```
Comment (TomasTomecek) [inline docs/configuration/actions.md:133]: why not scissors? :(

`8<`

**Review by TomasTomecek — APPROVED** [review]
Real world example would be amazing, e.g. for the output mentioned in the docs - list commands that allow achieve that

File: docs/configuration/actions.md [hunk 2]
```diff
@@ -152,15 +187,23 @@ There are also action-specific environment variables:
 `PACKIT_PROJECT_VERSION` — version to be set in the specfile, set when relevant 
 (e.g. when syncing upstream release downstream)
 
+**commit-message**
+
+`PACKIT_UPSTREAM_TAG` - git tag of the upstream release
+`PACKIT_UPSTREAM_COMMIT` - commit SHA of the upstream release
+
```
Comment (lbarcziova) [inline docs/configuration/actions.md:194]: isn't `PACKIT_PROJECT_VERSION` missing?

Comment (mfocko) [reply docs/configuration/actions.md]: oh right /o

File: docs/configuration/actions.md (hunk 1 above)
Comment (mfocko) [reply docs/configuration/actions.md:139]: it's almost the same :)

**Comment by mfocko** [comment]
added the example for the example output

**Comment by mfocko** [comment]
also reformatted the environment variables

File: docs/configuration/actions.md [hunk 3]
```diff
@@ -123,43 +124,108 @@ actions:
...
 ## Environment variables set by packit
 
 Additionally, packit sets several environment variables for the actions:
 
-`PACKIT_UPSTREAM_PROJECT_NAME` — set to the `upstream_package_name` value, if any  
-`PACKIT_DOWNSTREAM_PROJECT_NAME` — set to the `downstream_package_name` value, if any  
-`PACKIT_CONFIG_PACKAGE_NAME` — set to the package name key for the `packages` dictionary in a monorepo project,
+* `PACKIT_UPSTREAM_PROJECT_NAME` — set to the `upstream_package_name` value, if any
+* `PACKIT_DOWNSTREAM_PROJECT_NAME` — set to the `downstream_package_name` value, if any
+* `PACKIT_CONFIG_PACKAGE_NAME` — set to the package name key for the `packages` dictionary in a monorepo project,
```
Comment (lbarcziova) [inline docs/configuration/actions.md:186]: just for completeness, I would add `PACKIT_DEBUG_DIVIDER` here as well

Comment (mfocko) [reply docs/configuration/actions.md:186]: It’s below with the commit-message, it’s a bit confusing though

Comment (mfocko) [reply docs/configuration/actions.md:186]: Where did I lose it? 🙃

File: docs/configuration/actions.md [hunk 4]
```diff
@@ -123,43 +124,108 @@ actions:
...
+    - Resolves rhbz#1234
+
+This output can be produced by the following config:
+```yaml
+actions:
+  commit-message:
+    - echo 'debug output'
+    - bash -c 'echo ${PACKIT_DEBUG_DIVIDER}'
+    - bash -c 'echo -e "Rebase to new upstream release ${PACKIT_PROJECT_VERSION}\n"'
+    - echo '- Resolves rhbz#1234'
```
Comment (TomasTomecek) [inline docs/configuration/actions.md:165]: sweet, exactly what I was looking for

**Review by TomasTomecek — APPROVED** [review]

File: docs/configuration/actions.md (hunk 1 above)
Comment (mfocko) [reply docs/configuration/actions.md:139]: btw I'm willing to change it, if you have a font that provides some fancy ligature for this :P

**Pull request merged**